prow/config/tide_test.go

```diff
@@ -54,6 +56,385 @@ var expectedQueryComponents = []string{
 	"review:approved",
 }
 
+func TestMarshalMergeMethod(t *testing.T) {
+	testCases := []struct {
+		testName   string
+		tideConfig TideGitHubConfig
+		wantYaml   string
+	}{
+		{
+			testName: "Org-wide",
```

Comment:
I'd like to see more testcases here, especially for:

- Shorthands
- Multiple matching items